README.rst

```diff
@@ -1,5 +1,10 @@
-OpenAstronomy Packaging Guide
-=============================
+SunPy Package Template
+======================
 
-The `OpenAstronomy Python Packaging Guide <https://packaging-guide.openastronomy.org/en/latest/>`__ that contains an embedded `cookiecutter <https://cookiecutter.readthedocs.io/>`__ template.
-Please see also the `instructions for using the template <https://packaging-guide.openastronomy.org/en/latest/#using-the-template>`__.
+This repo extends the `OpenAstronomy Python Packaging Guide <https://packaging-guide.openastronomy.org/en/latest/>`__ to add SunPy specific features such as config files which are shared over all of SunPy's packages.
+
+Previewing Rendered Templates
+-----------------------------
+
+To make it easier to preview changes to files when rendering the template there is a tox environment named ``bake_cookies``.
+If you run ``tox -e bake_cookies`` a new directory named ``cookies`` will be present with a variety of test renders of the template in (see ``tests/conftest.py`` for the configuration options).
```

Comment:
Does the ``--cookie-location`` work if that is passed to tox?

I ask as I have a similar problem with magex

Reply:
It is just a pytest CLI arg so it passes through like normal.

Reply:
Sure, but does it get parsed correctly. In the magex case, pytest says its not valid.